webpack.config.js

```diff
@@ -64,15 +65,7 @@ const webpackConfig = {
 			}
 		]
 	},
-	resolve: {
```

Comment:
Does this need to be moved to a separate config? Wouldn't it be enough to resolve using the `webpack.config.js` file from `.eslint-resolver.js` as-is?